src/execution_plan/ops/op_merge.c

```diff
@@ -5,7 +5,8 @@
 */
 
 #include "op_merge.h"
-#include "RG.h"
+#include "../../RG.h"
```

Comment:
`#include "RG.h"` should be fine as use `-I .` see Makefile line 124

Reply:
That construction is fine, but I prefer to use relative paths because my linter can't infer `-I` directives, so I get a false warning on this line.

Do you have a strong preference?